hooks/project-relman/code-review-production.yml

```diff
@@ -39,12 +39,21 @@ payload:
       - $if: firedBy == 'pulseMessage'
         else: {}
         then:
-          TRY_RUN_ID:
-            $eval: payload.runId
-          TRY_TASK_GROUP_ID:
-            $eval: payload.status.taskGroupId
-          TRY_TASK_ID:
-            $eval: payload.status.taskId
+          $if: '"runId" in payload && "status" in payload'
+          then:
+            # Triggered by code-review task in try ending
+            # so we can analyze a try patch
+            TRY_RUN_ID:
+              $eval: payload.runId
+            TRY_TASK_GROUP_ID:
+              $eval: payload.status.taskGroupId
+            TRY_TASK_ID:
+              $eval: payload.status.taskId
+
+          else:
+            # Triggered by end of build on autoland & MC
```

Comment:
I'm confused by this comment.  What about other l3 branches besides autoland and central?

Reply:
Other branches would trigger it as well, they are skipped in the bot's code 

I can update the comment to mention that.